addons/account/partner.py

```diff
@@ -252,12 +254,21 @@ def _journal_item_count(self, cr, uid, ids, field_name, arg, context=None):
         MoveLine = self.pool('account.move.line')
         AnalyticAccount = self.pool('account.analytic.account')
         results = {}
-        for partner_id in ids:
-            results[partner_id] = {}
-            if 'contracts_count' in field_name:
-                results[partner_id]['contracts_count'] = AnalyticAccount.search_count(cr, uid, [('partner_id', '=', partner_id)], context=context)
-            if 'journal_item_count' in field_name:
-                results[partner_id]['journal_item_count'] = MoveLine.search_count(cr, uid, [('partner_id', '=', partner_id)], context=context)
+        for partner_id in partner_ids:
+            for field, getter in {
+                'contracts_count': lambda: (
+                    AnalyticAccount.search_count(
+                        cr,uid, [('partner_id', '=', partner_id)], context=context)
+                ),
+                'journal_item_count': lambda: (
+                    MoveLine.search_count(
+                        cr, uid, [('partner_id', '=', partner_id)], context=context)
+                )
+            }.items():
+                try:
+                    results.setdefault(partner_id, {})[field] = getter()
+                except Exception:
```

Comment:
lost with the rebase, will update( if you looked on the former changesets, yes im catching accesserror)